packages/x-components/src/components/modals/base-id-modal-open.vue

```diff
@@ -66,6 +70,35 @@ The component rendering content passed to the default slot and opening the modal
 </script>
 ```
 
+### Replacing the default button
+
+The component renders whatever element is passed, replacing the default button and exposing the
+function to open the modal with modalId `my-modal`.
+
+```vue
+<template>
+  <BaseIdModalOpen modalId="my-modal">
+    <template #opening-element="{ openModal }">
+      <ul>
+        <li @click="openModal">Open here</li>
+        <li>Not here</li>
+      </ul>
+    </template>
+  </BaseIdModalOpen>
+</template>
+
+<script>
+  import { BaseIdModalOpen } from '@empathyco/x-components';
+
+  export default {
+    name: 'BaseIdModalOpenTest',
+    components: {
+      BaseIdModalOpen
+    }
+  };
+</script>
+```
+
 ## Events
```

Comment:
Can you move this section to the top of the `docs`? Same in the `base-id-modal-close` file.

Reply:
Not sure if I follow, I see them at the same height both in this file and in `base-id-modal-close`.

Reply:
Yep, in both files this section should be at the top of the `docs`. See [HistoryQueriesSwitch](https://github.com/empathyco/x/blob/099121b27b08cb898652045661fbf92f81e1024e/packages/x-components/src/x-modules/history-queries/components/history-queries-switch.vue) for reference.

Reply:
Ah Ok, got you.